packages/esm-active-visits-app/src/active-visits-widget/active-visits.component.tsx

```diff
@@ -112,6 +116,19 @@ const ActiveVisitsTable = (props) => {
           </TableContainer>
         )}
       </DataTable>
+      <div>
+        <ActiveVisitsPagination
+          pageNumber={currentPage}
+          totalItems={activeVisits.length}
+          currentItems={results.length}
+          pageUrl={`$\{openmrsSpaBase}/home/`}
+          pageSize={pageSize}
+          onPageNumberChange={({ page, pageSize }) => {
+            setPageSize(pageSize);
```

Comment:
Is there a reason why both states are being set? I doubt both will change at the same time.

Reply:
Correct, I'll just fix it.